terraform/modules/load_balancer/main.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  values = [yamlencode(
+    {
+      controller = {
+        replicaCount = var.enhanced_mqtt_load_balancer.replicaCount
+        service = {
+          tcpPorts = [
+            for port_config in var.enhanced_mqtt_load_balancer.port_configs :
+            port_config
+          ]
+          type: "LoadBalancer"
+          enablePorts = {
+            http: false
+            https: false
```

Comment:
When two mqtt ports are specified e.g: 8883 and 443 - will https = false disable mqtt traffic on port 443?

Reply:
no, it will not